minetest/minetest
# Don't memset SEvent v2

supersedes #15344

## To do

This PR is  Ready for Review.

## How to test

1. check gcc and clang CI for warnings
2. ensure that <s>Minetest</s> Luanti still generally works
